website/docs/docs/getting-started/custom-ui.md

```diff
@@ -184,42 +183,6 @@ experience by modifying the `recompileDelay` value or by setting the
 />
 ```
 
-## Sandpack Runner
```

Comment:
I would maybe keep the docs about Sandpack Runner with a mention that it was deprecated and an example of how to use the Preview to get the same result. WDYT?

Reply:
Makes sense. But I would only add an example of how to use the Preview and leave to mention the deprecated component in the Migration Guide. I'll add to my to-do list to address all the documentation parts at once.